src/Summoner/Template.hs

```diff
@@ -298,24 +298,30 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
 
     -- create README template
     readme :: Text
-    readme =
-        [text|
-        # $repo
-
-        [![Hackage]($hackageShield)]($hackageLink)
-        [![$license license](${licenseShield})](${licenseLink})
-        $stackBadges
-        $travisBadge
-        $appVeyorBadge
-
-        $description
-        $endLine
-        |]
+    readme = T.intercalate "\n" $ filter (/="") 
+        [ ("# " <> repo <> "\n")
+        , hackage
+        , licenseBadge
+        , stackBadges
+        , travisBadge
+        , appVeyorBadge
+        , ("\n" <> description)
+        ]
```

Comment:
instead of `filter (/="")` and memptyIfFalse in each component you can do
```haskell
    [ "# " <> repo
    , ""
    ] 
 ++ [ "some one line text | conditionBool ]
...
```